src/Jackett.Common/Definitions/yggtorrent.yml

```diff
@@ -213,6 +213,7 @@
           "a[href*=\"/filmvidéo/documentaire/\"]" : 2181
           "a[href*=\"/filmvidéo/emission-tv/\"]" : 2182
           "a[href*=\"/filmvidéo/film/\"]" : 2183
+          "a[href*=\"/film-video/film/\"]" : 2145
```

Comment:
i don't think you'll find this in the torrent search results.
1. up to now, only the sub-categories have been used to flag the categories in the results
2. they have not used video in the past, but vidéo
So are you sure about this?

Reply:
I notice several torrents with this category like this one:
https://www2.yggtorrent.gg/torrent/film-video/film/340525-a+x+l+2018+multi+720p+nf+webrip+x264-pr3d4t0r

If you search for  "A-X-L" , all of them are in the film-video/film category.

I found out later there is more in the log:
film-video/animation
film-video/documentaire
etc...

and Jackett can't parse it since they are not in the case selectors, so the search engine is limited.

Do you confirm the same behaviour?
Thank you

Reply:
excellent. yes I can confirm your finding with a-x-l.
so its seems that there are Latin-alphabet alternates to some of the category/subcategory link names.

so would you please add any latin-alphabet hits you have discovered so far to the case block, and assign them the same subcategory number as their counterpart that are using basic-glyphs.

so for instance,
```yaml
[...]
          "a[href*=\"/filmvidéo/film/\"]" : 2183
          "a[href*=\"/film-video/film/\"]" : 2183
[...]
          "a[href*=\"/filmvidéo/documentaire/\"]" : 2181
          "a[href*=\"/film-video/documentaire/\"]" : 2181
[...]
```
Thank you.

Reply:
Ok, they all follow the same scheme. I think it's due to a recent emergency backup they did, not sure.
Is it good?
Thanks